scipy/signal/_signaltools.py

```diff
@@ -1852,10 +1852,51 @@ def medfilt2d(input, kernel_size=3):
     -----
     This is faster than `medfilt` when the input dtype is ``uint8``,
     ``float32``, or ``float64``; for other types, this falls back to
-    `medfilt`; you should use `scipy.ndimage.median_filter` instead as it is
-    much faster.  In some situations, `scipy.ndimage.median_filter` may be
+    `medfilt`. In some situations, `scipy.ndimage.median_filter` may be
```

Comment:
This line shouldn't be deleted here.

Reply:
Hi @ilayn,

If we keep it duplicated, why not reverse the order of the 2nd line?
The docs of `scipy.ndimage.median_filter` refer you to this function with uint8/float32/float64 and cval=0.
Quote:
"
For 2-dimensional images with ``uint8``, ``float32`` or ``float64`` dtypes
the specialised function `scipy.signal.medfilt2d` may be faster. It is
however limited to constant mode with ``cval=0``. 
"

Example:

Before:
You should use `scipy.ndimage.median_filter` instead as it is much faster.  
In some situations, `scipy.ndimage.median_filter` may be faster than this function.

After:
You should use `scipy.ndimage.median_filter` instead as it is much faster.  
In some situations, this function may be faster than `scipy.ndimage.median_filter`.

What do you think?